graphics/jasper/Portfile

```diff
@@ -41,6 +48,14 @@ variant docs description {build documentation} {
         port:texlive-fonts-recommended  \
         port:texlive-latex-recommended
 
-    configure.args-delete   -DJAS_ENABLE_DOC=false
-    configure.args-append   -DJAS_ENABLE_DOC=true
+    configure.args-replace  -DJAS_ENABLE_DOC=OFF \
+                            -DJAS_ENABLE_DOC=ON
 }
+
+variant heif description {Enable the use of the HEIF Library} {
+    depends_lib-append      port:libheif
+    configure.args-replace  -DJAS_ENABLE_LIBHEIF=OFF \
+                            -DJAS_ENABLE_LIBHEIF=ON
+}
```

Comment:
Josh, would you be interested in enabling experimental HEIC support too, when this variant is enabled?

Reply:
> Josh, would you be interested in enabling experimental HEIC support too, via this variant?

If so, the change is available via `jasper-devel`, for testing/validation:

https://github.com/macports/macports-ports/commit/3b0b08aff32f2b15c014714e54ad2126f355b0af